modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -10,52 +20,100 @@ If you do not need to review the JSON files that the `ccoctl` tool creates befor
 Otherwise, you can create the AWS resources individually.
 
 //to-do if possible: xref to modules/cco-ccoctl-creating-individually.adoc for `create the AWS resources individually`
+endif::aws[]
+ifdef::alibaba[]
+[id="cco-ccoctl-creating-at-once_{context}"]
+= Creating Alibaba Cloud RAM users with ccoctl
+
+You can use the `ccoctl alibabacloud create-ram-users` command to automate the creation of Alibaba Cloud RAM users and policies for each in-cluster component.
+endif::alibaba[]
 
 [NOTE]
 ====
-By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this location.
 ====
 
 .Prerequisites
 
 * Extract and prepare the `ccoctl` binary.
+ifdef::alibaba[]
+* Obtain the AccessKey (AK) of the RAM user you wish to use.
```

Comment:
AccessKey should have sufficient permissions as follows. Not sure how to best put it in docs.

```
ram:CreatePolicy
ram:GetPolicy
ram:CreatePolicyVersion
ram:DeletePolicy
ram:DetachPolicyFromUser
ram:ListPoliciesForUser
ram:AttachPolicyToUser
ram:CreateUser
ram:GetUser
ram:DeleteUser
ram:CreateAccessKey
ram:ListAccessKeys
ram:DeleteAccessKey
```

Reply:
Michael has that detail in his PR: https://github.com/openshift/openshift-docs/pull/41083/files#diff-ea32beb8c4e76d3c5d0f0a1d943a4c7f2de05d4687864350944730eddf575dea
That section should appear ahead of this one as a pre-req.